.github/workflows/build_and_test.yml

```diff
@@ -204,3 +204,37 @@ jobs:
           name: ${{ runner.os }}-Unit-Tests
           path: Test*.xml
     if: ${{ always() }}
+
+  package:
+    name: Package
+    needs: [build_module, test_on_windows_v5, test_on_windows_v7, test_on_ubuntu, test_on_macos, test_against_cloud]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download
+        uses: actions/download-artifact@v3
+        with:
+          name: Release
+          path: ./Release/
+      - name: Setup PowerShell module cache
```

Comment:
What is it that we are caching here?
I skimmed over the documentation of `action/cache` (https://github.com/actions/cache). I didn't find a powershell example.

Is it "just" making the `Setup` faster?
do we need to change how `Setup` installs packages so it supports the caching?

Reply:
If this is just copy/paste from the other jobs, ignore my question

Reply:
The caching is to speed up between the different test environments. Loading dependency modules etc.

Nothing was changed here.